models.js

```diff
@@ -28,4 +28,23 @@ function getArtcileById(article_id) {
         return result.rows[0];
     });
 }
-module.exports={getTopics, getAllEndPoints, getArtcileById}
\ No newline at end of file
+
+
+function getArticleAndSort(sort_by = 'created_at', order = 'desc'){
```

Comment:
queries aren't strictly necessary for this ticket (you could just hardcode in the default sort by), but the way you've done it looks great so far